### PR TITLE
PVAS-462: update googlepay applepay specs

### PR DESCRIPTION
**Pull request by rich-cooper-cko**
# Description of changes in PR

[//]: # 'Adding `token_format` to googlepay and applepay tokens response'

## Checklist

- [x] Added a changelog entry to the `changelog.md` file in either `abc_spec` or `nas_spec`
- [ ] Contacted the Tech Docs team about any corresponding guides that need to be updated for www.checkout.com/docs

**Contributing options**: Look at [our documentation](https://checkout.atlassian.net/wiki/spaces/PD/pages/2169506663/API+ref+publication+process) for options to contribute to the API reference.

### CODE REVIEW

**Review by sebastian-garcia-cardona-cko — APPROVED** [review]

File: nas_spec/changelog.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 | Date       | Description of change                                                                                                 |
 | ---------- | --------------------------------------------------------------------------------------------------------------------- |
+| 2022/08/03 | Add missing `token_format` to googlepay and applepay token responses                                                  |
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/changelog.md:5]:
```suggestion
| 2022/08/03 | Add missing `token_format` to Google Pay and Apple Pay token responses.                                              |
```

File: abc_spec/changelog.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 | Date       | Description of change                                                                                           |
 |------------|-----------------------------------------------------------------------------------------------------------------|
+| 2022/08/03 | Add missing `token_format` to googlepay and applepay token responses                                            |
```
Comment (sebastian-garcia-cardona-cko) [inline abc_spec/changelog.md:5]:
```suggestion
| 2022/08/03 | Add missing `token_format` to Google Pay and Apple Pay token responses.                                            |
```

File: abc_spec/components/schemas/Tokens/01_ApplePayTokenResponse.yaml [hunk 3]
```diff
@@ -2,3 +2,11 @@ type: object
 description: Apple Pay Token Response
 allOf:
   - $ref: '#/components/schemas/TokenResponse'
+  - type: object
+    properties:
+      token_format:
+        type: string
+        description: The format of the token
```
Comment (sebastian-garcia-cardona-cko) [inline abc_spec/components/schemas/Tokens/01_ApplePayTokenResponse.yaml:9]:
```suggestion
        description: The format of the token.
```

File: abc_spec/components/schemas/Tokens/02_GooglePayTokenResponse.yaml [hunk 4]
```diff
@@ -2,3 +2,12 @@ type: object
 description: Google Pay Token Response
 allOf:
   - $ref: '#/components/schemas/TokenResponse'
+  - type: object
+    properties:
+      token_format:
+        type: string
+        description: The format of the token
```
Comment (sebastian-garcia-cardona-cko) [inline abc_spec/components/schemas/Tokens/02_GooglePayTokenResponse.yaml:9]:
```suggestion
        description: The format of the token.
```

File: nas_spec/components/schemas/Tokens/02_GooglePayTokenResponse.yaml [hunk 5]
```diff
@@ -1,4 +1,14 @@
 type: object
 description: Google Pay Token Response
-allOf:
+allOf: 
   - $ref: '#/components/schemas/TokenResponse'
+  - type: object
+    properties:
+      token_format:
+        type: string
+        description: The format of the token
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/components/schemas/Tokens/02_GooglePayTokenResponse.yaml:9]:
```suggestion
        description: The format of the token.
```

File: nas_spec/components/schemas/Tokens/03_ApplePayTokenResponse.yaml [hunk 6]
```diff
@@ -2,3 +2,11 @@ type: object
 description: Apple Pay Token Response
 allOf:
   - $ref: '#/components/schemas/TokenResponse'
+  - type: object
+    properties:
+      token_format:
+        type: string
+        description: The format of the token
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/components/schemas/Tokens/03_ApplePayTokenResponse.yaml:9]:
```suggestion
        description: The format of the token.
```

**Review by sebastian-garcia-cardona-cko — APPROVED** [review]

**Review by sebastian-garcia-cardona-cko — APPROVED** [review]

**Review by Joshua-Bedeau-cko — COMMENTED** [review]
Looks good to me but this is a token endpoint can Vault Product verify if needed?

**Review by jenny-dosti-cko — DISMISSED** [review]

**Review by jenny-dosti-cko — APPROVED** [review]

**Review by sebastian-garcia-cardona-cko — APPROVED** [review]

**Pull request merged**